Lib/glyphsLib/classes.py

```diff
@@ -3090,12 +3095,32 @@ def __init__(self):
         self.visible = True
         self.weight = self._defaultsForName["weightClass"]
         self.width = self._defaultsForName["widthClass"]
+        self.type = self._defaultsForName["type"]
 
     customParameters = property(
         lambda self: CustomParametersProxy(self),
         lambda self, value: CustomParametersProxy(self).setter(value),
     )
 
+    @property
+    def type(self):
+        if self._type == INSTANCETYPEVARIABLE:
+            return "variable"  # "variable" is stored in plist
```

Comment:
it's weird this `type` property returns either a constant string literal "variable" or a False bool value. What does Glyphs.app scripting API do in this case?

Reply:
> the plist the value is stored as str "variable" or not at all

then the conversion from `1` to `"variable"` should be done inside `_serialize_to_plist` method before writing the `type` key